### ISSUE TITLE
test failures in Python 3.7

### ISSUE THREAD

**Issue by carlwgeorge** [maxmind/MaxMind-DB-Reader-python#35]
The test suite passes for me with Python 3.6.5, but when run with Python 3.7.0b5 there are two failures.

```
======================================================================
FAIL: test_no_constructor_args (reader_test.TestAutoReader)
----------------------------------------------------------------------
TypeError: function missing required argument 'database' (pos 1)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/maxminddb-1.4.0/tests/reader_test.py", line 179, in test_no_constructor_args
    cls)
AssertionError: " 1 required positional argument|\(pos 1\) not found|takes at least 2 arguments" does not match "function missing required argument 'database' (pos 1)"

======================================================================
FAIL: test_no_constructor_args (reader_test.TestExtensionReader)
----------------------------------------------------------------------
TypeError: function missing required argument 'database' (pos 1)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/maxminddb-1.4.0/tests/reader_test.py", line 179, in test_no_constructor_args
    cls)
AssertionError: " 1 required positional argument|\(pos 1\) not found|takes at least 2 arguments" does not match "function missing required argument 'database' (pos 1)"

----------------------------------------------------------------------
```

**Comment by oschwald** [comment]
Thanks for reporting! We will take a look at this.

**Comment by carlwgeorge** [comment]
Thanks for the quick fix and release!

@hroncok This fix is in [version 1.4.1](https://github.com/maxmind/MaxMind-DB-Reader-python/releases/tag/v1.4.1).  I [updated python-maxminddb's master branch](https://src.fedoraproject.org/rpms/python-maxminddb/c/e5916d74600aa770437a3a5e9b9c5e80c4e4a602) with that version, so you should be clear to rebuild it for Python 3.7 now.
